model/src/main/java/uk/gov/hmcts/payment/api/service/AccountServiceImpl.java

```diff
@@ -1,31 +1,48 @@
 package uk.gov.hmcts.payment.api.service;
 
-import io.github.resilience4j.circuitbreaker.annotation.CircuitBreaker;
 import io.github.resilience4j.timelimiter.annotation.TimeLimiter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import io.github.resilience4j.circuitbreaker.annotation.CircuitBreaker;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Profile;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
 import org.springframework.stereotype.Service;
+import org.springframework.web.client.ResourceAccessException;
 import org.springframework.web.client.RestTemplate;
 import uk.gov.hmcts.payment.api.dto.AccountDto;
 import uk.gov.hmcts.payment.api.util.AccountStatus;
 
 import java.math.BigDecimal;
-import java.util.Date;
+import java.util.concurrent.TimeoutException;
 
 @Service
 @Profile("!liberataMock")
 public class AccountServiceImpl implements AccountService<AccountDto, String> {
 
+    private static final Logger LOG = LoggerFactory.getLogger(AccountService.class);
+
+    @Autowired
+    private LiberataService liberataService;
+
+    @Autowired
+    @Qualifier("restTemplateLiberata")
     private RestTemplate restTemplate;
 
     @Value("${liberata.api.account.url}")
     private String baseUrl;
 
     @Override
     @CircuitBreaker(name = "defaultCircuitBreaker")
-    @TimeLimiter(name = "retrievePbaAccountTimeLimiter")
-    public AccountDto retrieve(String pbaCode) {
-        if(pbaCode.equalsIgnoreCase("PBAFUNC12345")){
+    // @TimeLimiter(name = "retrievePbaAccountTimeLimiter")
```

Comment:
I will remove retrievePbaAccountTimeLimiter from application.properties since this is the only place it's being used.

Reply:
Thanks Sammi.